src/scripts/DropdownButton.js

```diff
@@ -77,11 +77,9 @@ export default class DropdownButton extends Component {
 
   onMenuItemClick(...args) {
     if (!this.props.hoverPopup) {
-      setTimeout(() => {
-        const triggerElem = ReactDOM.findDOMNode(this.trigger);
-        if (triggerElem) triggerElem.focus();
-        if (triggerElem) this.setState({ opened: false });
-      }, 10);
+      const triggerElem = ReactDOM.findDOMNode(this.trigger);
```

Comment:
check all the places that use this component in VisualForceCore that this change doesn't break their functionality.
The places that i know that use this component is in top of the comment text and in my Work -> the new button.
search for other places and check that they are working.